git_repo/repo.py

```diff
@@ -282,7 +276,7 @@ def do_fork(self):
         else:
             # git <target> fork <user>/<repo>
             if not self.target_repo:
-                if not self.user_name:
+                if not self.namespace:
                     raise ArgumentError('Cannot clone repository, '
                                         'you shall provide either a <user>/<repo> parameter '
```

Comment:
don't forget to change `<user>/<repo>` as well in the `__doc__` at the top of the file (the `--help`) and on other occurences like here! 😉